src/modules/unit/helpers.js

```diff
@@ -249,15 +248,7 @@ const _sortByDistance = (
   });
 };
 
-export const sortByDistance = memoize(
-  _sortByDistance,
-  (units, pos, leafletMap, filterString) => {
-    if (leafletMap === null || units.length === 0 || pos === undefined) {
-      return '0';
-    }
-    return `${filterString};${pos[0]};${pos[1]}`;
-  }
-);
+export const sortByDistance = _sortByDistance;
```

Comment:
This memoization cached the result of an empty search as the last most search with a value.

I'm not sure what the original purpose of this optimization was. I tinkered with it, but couldn't understand what utility it provided.

I removed it entirely and checked manually that there was no obvious performance hit. As far as I tested the application, performance wasn't impacted. However, the application is so slow in general that even bigger changes may be a bit hard to detect.